packages/bogue/bogue.1907.11/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
```
You don't seem to use dune substitutions

Reply:
thanks a lot for you work on this package. I will apply your suggestions. (changing the version will change the tag, hence I suppose I have to open another PR?)

A question about SDL: in fact for systems like Ubuntu 14 or 16, they have SDL < 2.0.6, which prevents tsdl from compiling; but my library will still work if they use tsdl.0.9.2 instead of the current tsdl. Is there a way to implement such a test in the package?

Reply:
I'm not sure how to remove this, the opam file is generated by dune

Reply:
You are allowed to edit the file on your own.

Reply:
It's ok if you don't do it. It's not the end of the world